### PR TITLE
Added Pluto.jl to Browser workflow in docs

### LINKED ISSUE #44640: Add Pluto.jl to the Browser-Based Workflow
I suggest adding Pluto.jl to the browser-based workflow: https://docs.julialang.org/en/v1/manual/workflow-tips/#Browser-based-workflow

### PR DESCRIPTION
**Pull request by SamarthMayya**
Closes #44640

### CODE REVIEW

**Comment by SamarthMayya** [comment]
@simeonschaub, any chance you could review this Pull Request😅

File: doc/src/manual/workflow-tips.md [hunk 1]
```diff
@@ -64,8 +64,13 @@ line. A common pattern includes the following elements:
...
 
-It is also possible to interact with a Julia REPL in the browser via [IJulia](https://github.com/JuliaLang/IJulia.jl).
-See the package home for details.
+Here are a few ways to interact with the Julia REPL in a browser:
+
+  * One way is via [IJulia](https://github.com/JuliaLang/IJulia.jl). 
+    See the package home for details.
+
+  * Another way is via [Pluto](https://github.com/fonsp/Pluto.jl) notebooks. 
+    See the [wiki](https://github.com/fonsp/Pluto.jl/wiki) for details.
```
Comment (rikhuijzer) [inline doc/src/manual/workflow-tips.md:73]: Fons is actually working on a new website so this comment will become outdated pretty soon. Maybe remove both suggestion for details and just link to the core GitHub page?

Comment (SamarthMayya) [reply doc/src/manual/workflow-tips.md]: @rikhuijzer done, is it fine now?

File: doc/src/manual/workflow-tips.md [hunk 2]
```diff
@@ -64,8 +64,11 @@ line. A common pattern includes the following elements:
 
 ## Browser-based workflow
 
-It is also possible to interact with a Julia REPL in the browser via [IJulia](https://github.com/JuliaLang/IJulia.jl).
-See the package home for details.
+Here are a few ways to interact with the Julia REPL in a browser:
+
+  * One way is via [IJulia](https://github.com/JuliaLang/IJulia.jl).
+
+  * Another way is through [Pluto](https://github.com/fonsp/Pluto.jl) notebooks.
```
Comment (rikhuijzer) [inline doc/src/manual/workflow-tips.md:71]:
```suggestion
There are a few ways to interact with Julia in a browser:
- [IJulia.jl](https://github.com/JuliaLang/IJulia.jl)
- [Pluto.jl](https://github.com/fonsp/Pluto.jl)
```

Comment (rikhuijzer) [reply doc/src/manual/workflow-tips.md:71]: Similar now to the list in `Profile`: https://raw.githubusercontent.com/JuliaLang/julia/master/doc/src/manual/profile.md

Comment (SamarthMayya) [reply doc/src/manual/workflow-tips.md]: @rikhuijzer I've made those changes. Is it fine now?

**Review by rikhuijzer — APPROVED** [review]

File: doc/src/manual/workflow-tips.md (hunk 2 above)
Comment (SamarthMayya) [reply doc/src/manual/workflow-tips.md]: @rikhuijzer any idea about when this MR will be merged?

**Comment by ViralBShah** [comment]
@SamarthMayya Welcome and thanks for your contribution to Julia.

**Pull request merged**